app/views/trips/_form.html.erb

```diff
@@ -11,27 +11,35 @@
     </div>
   <% end %>
 
+
+  <form>
```

Comment:
I believe `form_with` from line 1 will add these `form` tags for you, so you could remove `<form>` and `</form>`